apps/testing-javascript/src/data/sanity-import.ts

```diff
@@ -0,0 +1,559 @@
+// Running this script:
+//
+// Ensure you have `dataFilePath` set to the location of the formatted export
+// of Testing JavaScript data. Also, be sure to set `SANITY_EDITOR_TOKEN` in
+// your `.env.local` environment variables file.
+//
+// Execute the Mux Asset Upload script with:
+//
+// ```
+// npx ts-node --skipProject src/data/sanity-import.ts
+// ```
+
+import fs from 'fs'
+import fetch from 'node-fetch'
+// import sanityClient from '@sanity/client'
+import {v4 as uuidv4} from 'uuid'
+import {z} from 'zod'
+import last from 'lodash/last'
+
+require('dotenv-flow').config({
+  default_node_env: 'development',
+})
+
+const {
+  NEXT_PUBLIC_SANITY_PROJECT_ID: PROJECT_ID,
+  NEXT_PUBLIC_SANITY_DATASET: DATASET,
+  SANITY_EDITOR_TOKEN: EDITOR_TOKEN,
+  NEXT_PUBLIC_SANITY_API_VERSION: API_VERSION,
+  SANITY_IMPORT_DATA_FILE_PATH: dataFilePath,
+} = process.env
+
+const muxUploadPath = './mux-upload.json'
+
+function sleep(ms: number) {
+  return new Promise((resolve) => {
+    setTimeout(resolve, ms)
+  })
+}
+
+const SanitySlugSchema = z.object({
+  _type: z.literal('slug'),
+  current: z.string(),
+})
+type SanitySlug = z.infer<typeof SanitySlugSchema>
+
+const SanityBlockSchema = z.array(
+  z.object({
+    style: z.literal('normal'),
+    _type: z.literal('block'),
```

Comment:
There are a bunch of Zod schemas in this file that represent various elements of our Sanity schema -- these are a great code artifact that others can reference, but I'm also hoping they can be extracted and shared at some point with other scripts that are interacting with the Sanity API for this schema.